### DISCUSSION TITLE
Unknown publisher - installation stopped by a corporate policy.

### DISCUSSION THREAD

**Discussion by blonDIE78** [valinet/ExplorerPatcher#3298] [General]
Hello

I know it's silly but I am no developer and can't do it by myself. My corporate policy has changed and I can' t any longer install any unsigned software, 
Would you please release  a digitally signed .exe file ?

Thank you in advance.

**Comment by pyrates999** [comment]
This has been discussed here: https://github.com/valinet/ExplorerPatcher/discussions/2353

You also should not be installing this on any work computer since IT can take away your administrator privileges at any time, and the you could no longer install/uninstall or even update EP then.
